symfony/dependency-injection/CVE-2019-10910.yaml

```diff
@@ -0,0 +1,35 @@
+title:     "CVE-2019-10910: Check service IDs are valid"
+link:      https://symfony.com/cve-2019-10910
+cve:       CVE-2019-10910
+branches:
+    2.7.x:
+        time:     2019-04-16 12:48:27
+        versions: ['>=2.7.0', '<2.7.51']
+    2.8.x:
+        time:     2019-04-16 12:48:27
+        versions: ['>=2.8.0', '<2.8.50']
+    3.0.x:
+        time:     2019-04-16 12:48:27
```

Comment:
time should be null, as that branch is not patched (same for all other unpatched branches)